mscore/pagesettings.h

```diff
@@ -22,29 +22,42 @@
 #define __PAGESETTINGS_H__
 
 #include "ui_pagesettings.h"
+#include "libmscore/mscore.h"
+#include "libmscore/page.h"
 
 namespace Ms {
 
 class MasterScore;
 class Score;
 class Navigator;
 
+// For converting from Ms::Units to a two-char string used as a suffix
+static const char* unitSuffixes[] = {"px", "mm", "in", "sp"};
```

Comment:
I tried that, and a bunch of variations on it, and the compiler choked every time except once.  And that one version didn't store the strings, it was all empty, so the memory wasn't being allocated, or something...
Is there a global, single instance of some class that I can use to stash this?  I could make it a property of that class.